### PR TITLE
Add compat data for ImageData

### PR DESCRIPTION
**Pull request by maboa**

### CODE REVIEW

File: api/ImageData.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "ImageData": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ImageData/ImageData",
```
Comment (Elchi3) [inline api/ImageData.json]: Please add a description that clarifies this is the constructor.
 `"description": "<code>ImageData()</code> constructor",`

File: api/ImageData.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+            "version_added": "3.2"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "Support in workers": {
```
Comment (Elchi3) [inline api/ImageData.json]: No spaces in identifiers. Please name this `worker_support` and add `"description": "Support in workers",`.

Comment (maboa) [reply api/ImageData.json]: 👍

File: api/ImageData.json (hunk 1 above)
Comment (maboa) [reply api/ImageData.json]: :+1:

**Pull request merged**